keyboards/latinpad/config.h

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 Latincompass@latincompass
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.*/
+
+#pragma once
+
+#include "config_common.h"
+#define VENDOR_ID       0x4444 // "ww" = ww
+#define PRODUCT_ID      0x5350 // "ri" = rich
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    latincompass
+#define PRODUCT         latinpad
+#define DESCRIPTION     dual rotary encoder macro pad
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 4
+#define MATRIX_ROW_PINS { F4, F5, F6, F7, B1 }
```

Comment:
```suggestion
#define MATRIX_ROW_PINS { B1, F4, F5, F6, F7 }
```

Reply:
This will move the current last row in your layout macro to be the first row instead.

Reply:
THANKS sir,But if do this the last rows ELCEctric will be wrong

Reply:
I'm not sure that it will be.  It's more of the order that the pins are scanned, and I think you should be fine. 

It should be simple to test and verify, though.